### PR TITLE
Add tasks

### PR DESCRIPTION
**Pull request by Manpreet-hub**
- Added  Modal and Context (implemented CRUP operations)
- Added task list where all created tasks will be shown.

chrome_u0kaKKHTuI

### CODE REVIEW

File: src/components/Modal/Modal.js [hunk 1]
```diff
@@ -1,5 +1,144 @@
...
+//           }))
+//         }
+//       ></textarea>
+//       <input
+//         className="input input-fields"
+//         type="number"
+//         placeholder="Add Time in Mintues"
+//         value={taskDetails.time}
+//         onChange={(e) =>
+//           setTaskDetails((prevData) => ({
```
Comment (ashishnnnnn) [inline src/components/Modal/Modal.js]: Remove this commented code..

**Review by ashishnnnnn — APPROVED** [review]
Overall code looks good to merge :+1:

File: src/components/Modal/Modal.js (hunk 1 above)
Comment (Manpreet-hub) [reply src/components/Modal/Modal.js]: yes, removed the commented code

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean I liked that you have created a separate file for constant

**Pull request merged**